cli/src/cmd/forge/create.rs

```diff
@@ -95,9 +96,11 @@ impl CreateArgs {
             compile::compile(&project, false, false)
         }?;
 
+        // output.slash_paths();
```

Comment:
should this not be commented?

Reply:
forgot to delete, no longer required as output is now slashed by default